src/util.hpp

```diff
@@ -496,8 +496,7 @@ inline bool contains(const Container & container, const Value & value)
 
 
 #if 1
-# include <SDL_types.h>
-typedef Sint32 fixed_t;
+typedef int32_t fixed_t;
```

Comment:
Well, personally, I think a fixed point real number is better implemented as a class type (I even implemented it for the formula engine before giving up on that refactor for the time being). But you're right, this isn't really relevant to the PR.